sinonjs/sinon
# Remove npm run all

In 0d1e40b the last use of `run-p` was removed, which is arguably the whole justification for this package.

It is trivial to run commands *serially* using `npm run` and `&&`, making this package superfluous.

Removing it, reduces the maintenance burden slightly and will speed up package install
slightly.

 #### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm test`
4. Observe that the test command still works

 #### Checklist for author
